### PR TITLE
refactor: Apply Google Format 

### PR DESCRIPTION
**Pull request by dricazenck**
* Enabled Google Format
* Fix Checkstyle Setup
* Fix checkstyle warnings
* Fix wrong merge conflicts in README
* Improve Readme Setup 

Api's is working: 
image

All Testing are working: 
image

### CODE REVIEW

**Review by nverbic — APPROVED** [review]
Most of the changes are in style so I suggest to merge the changes. Merge conflicts are solved and JavaTimeModule is added.

File: src/main/java/com/wcc/platform/domain/platform/LeadershipMember.java [hunk 1]
```diff
@@ -7,14 +7,18 @@
 
 /**
  * Represents the core team of the community: {@link MemberType#DIRECTOR}, {@link MemberType#LEADER}
+<<<<<<< HEAD
```
Comment (nverbic) [inline src/main/java/com/wcc/platform/domain/platform/LeadershipMember.java:10]: Should you remove this HEAD section?

File: src/main/java/com/wcc/platform/domain/platform/Member.java [hunk 2]
```diff
@@ -5,6 +5,18 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+<<<<<<< HEAD
```
Comment (nverbic) [inline src/main/java/com/wcc/platform/domain/platform/Member.java:8]: Same here?

File: src/main/java/com/wcc/platform/domain/platform/SocialNetwork.java [hunk 3]
```diff
@@ -1,5 +1,10 @@
 package com.wcc.platform.domain.platform;
 
+<<<<<<< HEAD
```
Comment (nverbic) [inline src/main/java/com/wcc/platform/domain/platform/SocialNetwork.java:3]: ?

File: src/main/java/com/wcc/platform/domain/platform/LeadershipMember.java (hunk 1 above)
Comment (dricazenck) [inline src/main/java/com/wcc/platform/domain/platform/LeadershipMember.java:10]:
```suggestion
```

**Pull request merged**